src/main/java/euteam/cleanmachine/service/UserService.java

```diff
@@ -72,6 +74,18 @@ public UserDto addUser(UserSignUpDto userSignUpDto) {
         return userDto;
     }
 
+    public UserDto LinkCard(NFCDto nfcDto) {
+        AuthItem  authlist;
+       // NFCCard card = new NFCCard();
+      //  card.setCardNumber(nfcDto.getCardNumber());
+        //UserDto userDto = new UserDto();
+      //  userDto.setAuthItemList();

```

Comment:
Looks great, but you need to get the user entity from the database with the username passed as a paramter, add the NFC Card to it's `AuthItem` list, and then save it again into the database with the `UserDao`.
```java
public UserDto LinkCard(String username, NFCDto nfcDto) {
  // ...
} 
```